mgmtd/mgmt_testc.c

```diff
@@ -155,7 +155,7 @@ static void sigusr1(void)
 	zlog_rotate();
 }
 
-static void quit(int exit_code)
+static __attribute__((__noreturn__)) void quit(int exit_code)
```

Comment:
I've made a macro that wraps this, makes it a little less wordy?

Reply:
my quick grep of the code base just showed me __no_return__ in a bunch of places....  I stopped looking for anything beyond that.  What's the macro?

Reply:
yes, sorry, should have said: `FRR_NORETURN` in lib/compiler.h ...

Reply:
fixed in a new commit